### PR TITLE
x11-misc/ydotool: new package, add 1.0.4

### PR DESCRIPTION
**Pull request by zpuskas**
Signed-off-by: Zoltan Puskas <zoltan@sinustrom.info>

### CODE REVIEW

**Comment by zpuskas** [comment]
This is a similar to `xdotool`, except it also works with Wayland and other interfaces, not only with X.org.

File: x11-misc/ydotool/ydotool-1.0.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+SRC_URI="https://github.com/ReimuNotMoe/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND=""
+DEPEND="
+	${RDEPEND}
+	app-text/scdoc
```
Comment (thesamesam) [inline x11-misc/ydotool/ydotool-1.0.4.ebuild:19]: BDEPEND, and the rest of *DEPEND can go then

File: x11-misc/ydotool/ydotool-1.0.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+"
+
+src_compile() {
+		cmake_src_compile ydotool ydotoold
+}
+
+src_install() {
+		cmake_src_install ydotool ydotoold
```
Comment (thesamesam) [inline x11-misc/ydotool/ydotool-1.0.4.ebuild:27]: Why do you need these explicit targets?

**Comment by zpuskas** [comment]
* Removed extra functions
* Updated License
* Install OpenRC file

**Pull request closed without merging**